### ISSUE TITLE
10.13 failed during installation

### ISSUE THREAD

**Issue by mathguru** [syscl/M3800#72]
today I try to install 10.13 but failed during installation
when I use versobe mode, I get the following error:
service only ran for 0 seconds, pushing respawn out by 10 seconds.
any one successes in 10.13？ 

---update
I was told that was caused by setup error of clover, but I'm still working for it.

**Comment by spud00** [comment]
https://www.tonymacx86.com/threads/fix-window-server-service-only-ran-for-0-seconds-with-dual-gpu.233092/

**Comment by mathguru** [comment]
@syscl 
After quick looking at these tutorials by RehabMan from tonymacx86.com, (here thanks @spud00 for the helpful link). I found that it's time-consuming and tough job to disable the discrete graphic card K1100, especially for rookie like me. So I'm just waiting for your code update to support 10.13.
Since you updated clover recently with some improvements. As you mentioned, the 10.27 update provide “Added apfs.efi so that users can update from HFS+ to APFS flawslessly”, Can your code automatically support 10.13？
Hope for your reply.
Regards

**Comment by philmb3487** [comment]
Is there a simple way to disable the Nvidia chip in the Unlocked Options BIOS??
Otherwise I'll see what I can cook up to do that in ACPI.

**Comment by syscl** [comment]
Hi, I have just updated my Precision M3800 from 10.12 to 10.13 from App Store without any issues. Just use my deploy.sh then Nvidia Chip will be disabled silently.

syscl

**Comment by philmb3487** [comment]
I was hoping to do a fresh install and ..

although I suppose I could use the same ACPI files once they've been patched by deploy under 10.12.

**Comment by syscl** [comment]
Right, ACPI patches are the same. I first update using patched ACPI then it boots fine. Later on I run deploy.sh again to see if it cause some issues, still same result, it boots fine.

syscl

**Comment by mathguru** [comment]
@syscl Ok, I will try fresh install again.

**Comment by mathguru** [comment]
Today, I retry fresh install of 10.13.1 with latest release, but still failed at "service only ran for 0 seconds, pushing respawn out by 10 seconds.“
maybe update from 10.12 and fresh install of 10.13 are two completely different things.

**Comment by philmb3487** [comment]
@mathguru I got that every time I've installed 10.13 as well, but when I reboot the install then finished successfully, and the OS works well after that.

I've had a few issues with USB when coming out of suspend where the touch screen and my external Wifi stops working, but apart from that things are good. The internal HDD i've got is pretty slow and battery life is not great, and the nvidia chip is just something you have to forget.

**Comment by philmb3487** [comment]
Point upgrades are buggy as well .. the best luck i've had was downloading the combo updater DMG and then selecting the right boot option I got 10.13.1 installed as well.

**Comment by philmb3487** [comment]
For initial install you will need a patched DSDT to disable the NVIDIA chip...

That or if you can somehow prevent the nvidia kext from loading, that should work too, but I haven't tried it. either by blacklisting somehow, or deleting the kext from the nested dmg, which is a pain because it's on a read-only image and so on. The boot option to disable the web driver is not enough anymore.

**Comment by mathguru** [comment]
@sinetek thanks for your reply. When you mentioned “right boot option” in your post，do u mean clover boot setup？

**Comment by philmb3487** [comment]
Yeah in the Clover booter I can see about 2-3 drives, named like "Recovery" "root" and "prevault booter" or something like this. I don't know what the last one does, but it's new since 10.13

if boot fails i try the other options

**Comment by starfireprime** [comment]
So I gave the 10.13 upgrade on my 9530 via the app store a shot and almost made it all the way through successfully.  These were my steps:
1) Updated my M3800 git and reapplied to Sierra to make sure everything was cool.
2) Pre-emptively updated my CLOVER/kext in my EFI partition with the 10.13 directory from syscl's distro.
3) Downloaded High Sierra from the App Store and initiated the update via command line so I could keep the HFS file system.
4) Plugged in a monitor via HDMI and processed the reboots for the upgrade
5) High Sierra first boot successfully and logged in via HDMI/monitor.  
6) Applied Deploy.sh and appeared to be OK.
7) Rebooted and could not get past initial "Apple" screen on my laptop.  I get the Apple and progress bar.  Then a spinner in the middle of that, but it never progresses.  HDMI/external monitor remained blank/off.  Same for updated Recovery partition.

I booted in verbose mode and reached a point where the laptop screen would clear/blank with a square cursor in the upper left, and then clear/blank again, over and over.

So I restored my backup and would like to try again.  Any ideas where I went wrong?  Thanks!

**Comment by mathguru** [comment]
@starfireprime thanks bro
step 4 is new and important for me, I will try in the weekend.

**Comment by allesklarbeidir** [comment]
@syscl , I have some troubles updating from app store.. I might be able to fix that - but, is hibernation working for you? I've read somewhere that it's not working with HighSierra.

Thanks a lot for your work, for 10.12.x it has been working really well for me!


EDIT:
I managed to install 10.13.1 from AppStore and everything is working well!
Even hibernation is working and after wakeup everything still works.

However, what might also be of interest for others:
After installing everything and running Deploy.sh and rebooting I had full gpu (intel) acceleration but a flikery screen. I used SwitchResX to define HiDPI FullHD resolution with a refresh rate slightly below 60hz and I had to use the "--patch-pixelclock" option from [https://github.com/the-darkvoid/XPS9530-OSX](url) to fix that.
